util/dvsim/Deploy.py

```diff
@@ -378,17 +378,20 @@ def _set_attrs(self):
         if self.sim_cfg.args.build_timeout_mins is not None:
             self.build_timeout_mins = self.sim_cfg.args.build_timeout_mins
         if self.build_timeout_mins:
-            log.log(VERBOSE, "Compile timeout for job \"%s\" is %d minutes",
-                    self.name, self.build_timeout_mins)
+            log.debug(f"Timeout for job \"{self.name}\" is "
```

Comment:
It is encouraged to have log calls use printf-like "%", so it can completely skip generating the string if the log won't be emitted.